hetio/neo4j.py

```diff
@@ -71,7 +101,13 @@ def append(self, x):
     def create(self):
         if not self:
             return
-        self.db_graph.create(*self)
+
+# http://stackoverflow.com/questions/37676731/typeerror-create-takes-2-positional-arguments-but-4-were-given
+        if PY2NEO_VER == 3:
+            self.db_graph.create(union(self))
```

Comment:
Let's put 

```python
union = lambda v: reduce(or_, v)
```

under this if statement, since it's not really needed anywhere else